src/SFA.DAS.ApprenticeCommitments.Web/Helpers/HasDeliveryModelExtensions.cs

```diff
@@ -6,7 +6,7 @@ namespace SFA.DAS.ApprenticeCommitments.Web.Helpers
     public static class HasDeliveryModelExtensions
     {
         public static bool IsNormalDelivery(this IHaveDeliveryModel deliveryModel)
-            => deliveryModel.DeliveryModel == DeliveryModel.Normal;
+            => deliveryModel.DeliveryModel == DeliveryModel.Regular;
```

Comment:
Is it worthwhile changing these extension methods to IsRegularDelivery() and IsIrregularDelivery()

Reply:
Yes, good idea.  Done.